recipes/nng/all/conanfile.py

```diff
@@ -20,13 +20,15 @@ class NngConan(ConanFile):
         "nngcat": [True, False],
         "http": [True, False],
         "tls": [True, False],
+        "maxTaskqThreads": "ANY"
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "nngcat": False,
         "http": True,
         "tls": False,
+        "maxTaskqThreads": None
```

Comment:
```suggestion
        "max_taskq_threads": 16
```
https://github.com/nanomsg/nng/blob/v1.5.2/CMakeLists.txt#L127

Reply:
The problem with setting this to 16 is that if the nng team changes the the default value from 16 to another value in their cmake file, we still use 16 in the conanfile.

Reply:
that's not a problem. you can set any value according to the version. You can use `config_options` to set the default_option, according to the version.

Reply:
Not constantly, only when updating a version. We really care about reading source files when updating a version.

Reply:
OK, will do. But then the 'if' around the assignment needs to go, right? 
```
        if self.options.max_taskq_threads:
            self._cmake.definitions["NNG_MAX_TASKQ_THREADS"] = self.options.max_taskq_threads
```

Reply:
Yes. But it's mandatory always. There is no `if`, only `            self._cmake.definitions["NNG_MAX_TASKQ_THREADS"] = self.options.max_taskq_threads`

As you will validate it, it will be always a number. Also, 0 is a valid number according to the upstream.